sdk/eventgrid/eventgrid/README.md

```diff
@@ -10,7 +10,7 @@ Use the client library to:
 
 [Source code](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventgrid/eventgrid/) |
 [Package (NPM)](https://www.npmjs.com/package/@azure/eventgrid/v/next) |
-[API reference documentation](https://docs.microsoft.com/javascript/api/@azure/eventgrid/?view=azure-node-preview) |
+[API reference documentation](https://docs.microsoft.com/javascript/api/@azure/eventgrid/) |
```

Comment:
Have we confirmed that the new url does have updated content?

Reply:
I looked and they do.